process/factory/interceptorscontainer/baseInterceptorsContainerFactory.go

```diff
@@ -687,3 +687,46 @@ func (bicf *baseInterceptorsContainerFactory) generateHeartbeatInterceptor() err
 
 	return bicf.container.Add(identifierHeartbeat, interceptor)
 }
+
+// ------- ShardValidatorInfo interceptor
+
+func (bicf *baseInterceptorsContainerFactory) generateShardValidatorInfoInterceptor() error {
+	identifier := common.ConnectionTopic
+
+	shardValidatorInfoFactory, err := interceptorFactory.NewInterceptedShardValidatorInfoFactory(*bicf.argInterceptorFactory)
```

Comment:
maybe find another name here? NewInterceptedShardValidatorInfoFactory suggest that this component might be used on all "shard" nodes and we might have a "meta" implementation

Reply:
renaming is not complete: "shard" persists
Also we still have the `interceptedShardValidatorInfoFactory` struct & `generateShardValidatorInfoInterceptor` function